src/MAUI/Maui.Samples/Samples/GraphicsOverlay/SketchOnMap/SketchOnMap.xaml.cs

```diff
@@ -51,25 +64,37 @@ private void Initialize()
             // Set a viewpoint on the map view.
             MyMapView.SetViewpoint(new Viewpoint(64.3286, -15.5314, 72223));
 
-            // Fill the combo box with choices for the sketch modes (shapes).
+            // Fill the array with choices for the sketch modes (shapes).
```

Comment:
`_sketchModes = Enum.GetNames(typeof(SketchCreationMode));` can replace this loop.